assets/js/translator.js

```diff
@@ -132,6 +133,12 @@ if(modeEnabled('translation')) {
             }, timeout);
 
             synchronizeTextareaHeights();
+
+            if(isURL($('#originalText').val())) {
+                setTimeout(function () {
+                    translateWebpage();
```

Comment:
> website translation should fire without delay on typing e.g. "reddit.com" in the textbox

Um, you had mentioned this in a previous comment. So is it okay if we don't call the translation if checkbox is not checked? If yes, I'd remove the timer and keep it consistent with the other behaviour.